index.html

```diff
@@ -1,47 +1,55 @@
 <!DOCTYPE html>
-<html ang="en">
+<html lang="en">
 <head>
     <title>YangZhou</title>
-    <meta charset="ascii"
+    <meta charset="utf-8">
 </head>
 <body>
-<header>
-<nav>
-<ul>
-  <li><a href="/CSD120/index.html">home</a> </li>
-  <li><a href="/CSD120/links.html">Links</a></li>
-  <li><a href="/CSD120/glossary.html">glossary</a></li>
-</ul>
-</nav>
-</header>
-<main>
-<a href="http://node11.codenvy.io:42865//CSD120/jacky.html" target="_blank"><h1>heading1</h1></a>
-<h2>My Three Favorite ball games
-Basketball, football and tennis</h2>
-<ol> <li>Basketball</li> <li>football</li> <li>tennis</li> </ol>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="/CSD120/index.html">home</a> </li>
+                <li><a href="/CSD120/links.html">Links</a></li>
+                <li><a href="/CSD120/glossary.html">glossary</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <h1>heading1</h1>
 
-<p>I am Zhou Yang from China. I like computer ga
-mes and football very much.</p>
-My favorite football club is Manchester City, because their football is very beautiful.
-<h2>Haiku</h2>
-<p>An old silent pond...
-A frog jumps into the pond,
-splash! Silence again.</p>
+        <p>I am Zhou Yang from China. I like computer ga
+        mes and football very much.</p>
+        <p>My favorite football club is Manchester City, because their football is very beautiful.</p>
 
-<p>Autumn moonlight-
-a worm digs silently
-into the chestnut.</p>
+        <h2>My Three Favorite ball games
+        Basketball, football and tennis</h2>
+        <ol> 
+            <li>Basketball</li> 
+            <li>football</li> 
+            <li>tennis</li> 
+        </ol>
 
-<p>In the twilight rain
-these brilliant-hued hibiscus -
-A lovely sunset.</p>
-<p><i>by</i> <b>Matsuo Basho</b><br /> </p>
-</main>
-<footer>
-<hr>
-<blockquote cite="https://examples.yourdictionary.com/examples-of-haiku-poems.html"></blockquote>
-<p><a href="https://mail.google.com/mail/u/0/#drafts?compose=CllgCJZZzfCjtHpNpcHBvcthdvCSDZPvCltvtgwZXhWjKgZjLbGfnSBjvnMMddZXWzTqnnpHNVV"> © 2019YangZhou</a></p>
-<b>&ndash Matsuo Basho</b><br />
-</footer>
+        <h2>Haiku</h2>
+        <p>An old silent pond...<br>
+        A frog jumps into the pond,<br>
+        splash! Silence again.</p>
+
+        <p>Autumn moonlight-<br>
+        a worm digs silently<br>
+        into the chestnut.</p>
+
+        <p>In the twilight rain<br>
+        these brilliant-hued hibiscus -<br>
+        A lovely sunset.<br>
+        <i>by <b>Matsuo Basho</b></i></p>
```

Comment:
The whole author line was to be emphasized with the strong element nested inside